test/views/declaration/spec/ViewSpec.scala

```diff
@@ -42,5 +44,8 @@ trait ViewSpec extends PlaySpec with OneAppPerSuite with ViewValidator {
 
   def assertMessage(key: String, expected: String): Unit = messages(key) must be(expected)
 
+  def fakeJorneyRequest(choice: String): JourneyRequest[AnyContentAsEmpty.type] =
+    JourneyRequest(AuthenticatedRequest(fakeRequest, ExportsTestData.newUser("", "")), new Choice(choice))
```

Comment:
Found something similar but simpler: `private val journeyRequestMock = mock[JourneyRequest[_]]` in `DeclarationTypeViewSpec`